pyproject.toml

```diff
@@ -24,6 +25,8 @@ black = "^24.3.0"
 flake8 = "^7.0.0"
 semver = "^3.0.2"
 tbump = "^6.11.0"
+pytest-asyncio = "^0.23.7"
```

Comment:
@thegeronimo 
If we put these in the dev deps, then it isnt forced to be installed into libs that depend on this, so its less libs and smaller images etc